tests/serverpod_test_server/test_integration/test_tools/protocol_test.dart

```diff
@@ -60,6 +60,31 @@ void main() {
       expect(result, ['PositionalArg', 'NamedArg']);
     });
 
+    test(
+        'when calling method with positional and nullable named args when passing value then echoes args',
```

Comment:
Suggestion: Change description to: `when calling method with positional and nullable named args with named arg then echoes args.`.

Reply:
75fa6ce